explorer/src/main/java/com/scalar/client/tool/explorer/command/Validate.java

```diff
@@ -51,10 +52,18 @@
   public void run() {
     ExplorerExecutor executor =
         explorer -> {
+          StatusCode statusCode;
           try {
             for (String assetId : assetIds) {
-              explorer.validate(assetId);
-              System.out.println(assetId + " is not tampered");
+              statusCode = explorer.validate(assetId).getCode();
```

Comment:
```suggestion
              StatusCode statusCode = explorer.validate(assetId).getCode();
```

Reply:
Thank you for review comments.
Updated review changes in fafec79